filebeat/module/system/_meta/kibana/7/dashboard/Filebeat-new-users-and-groups.json

```diff
@@ -521,7 +521,7 @@
                         "query": {
                             "query_string": {
                                 "analyze_wildcard": true, 
-                                "query": "_exists_:system.auth.useradd"
```

Comment:
You should leave this `_exists_:system.auth.useradd` here, this object is still there and always reliably filled.

Grabbing `_exists_:user` will give you a much bigger data set (e.g. logins), even if this is otherwise limited to the system.auth fileset.

Reply:
ok, resetted.